src/hooks/index.js

```diff
@@ -1,15 +1,19 @@
 /* eslint-disable no-param-reassign */
 const {
 	Forbidden,
+	GeneralError,
 	NotFound,
 	BadRequest,
 	TypeError,
 } = require('@feathersjs/errors');
 const { authenticate } = require('@feathersjs/authentication');
 
+const { v4: uuidv4 } = require('uuid');
+
 const { Configuration } = require('@schul-cloud/commons');
 const _ = require('lodash');
 const mongoose = require('mongoose');
+const { result } = require('lodash');
```

Comment:
Codacy found an issue: ['result' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)